packages/lodestar-utils/src/misc.ts

```diff
@@ -5,6 +5,10 @@ export function objectToCamelCase(obj: object): object {
   if (Object(obj) === obj) {
     Object.getOwnPropertyNames(obj).forEach((name) => {
       const newName = camelcase(name);
+      // @ts-ignore
+      if (obj[newName]) {
```

Comment:
I don’t believe this will correctly check if the property already exists. I think we should have `obj.hasOwnProperty(newName)` instead.

I say this because with this case, if `obj[newName]` is falsey but still already exists, this error won’t get thrown.

Reply:
good call